src/index.html

```diff
@@ -11,7 +11,125 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+    method="post" 
+    autocomplete="off">
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input type="text" 
+          name="surname"
+          required>
```

Comment:
use correct indentations for whole document
```suggestion
          <input type="text" 
            name="surname"
            required
          >
```